contents/handbook/strategy/overview.md

```diff
@@ -24,7 +24,7 @@ _Provide every tool needed for evaluating feature success._ The best use of an e
 
 _Get in first._ By already being used by our customers, we’re the default for each additional tool they add. It’s the technical co-founder and early engineers building the MVP and integrating the first product tools, not PMs. By focusing on engineers we can be their preferred choice and get in first. Additionally, we can ladder our tools - for example, session recording is used much earlier in the life cycle of the product than others like the CDPs helping us get in earlier than competing products.
 
-_Be the pipeline for product and customer data._ Traditionally, as companies scale their data warehouse becomes the source of truth and non-warehouse native tools (like product analytics) become less relevant. By being their core pipeline from connecting their data to their warehouses we can remain sticky for the life of our customers. And by providing this infra, we ensure the data we have remains comprehensive.
+_Be the source of truth for data._ Traditionally, as companies scale their data warehouse becomes the source of truth and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data in them. However, by providing the data and data-intense tools in one place, we can enhance the power of our products (like product analytics), provide increased trust, _and_ enable companies to build on top of the warehouse itself as they see fit, all _without_ them having to setup a complex stack.
```

Comment:
```suggestion
_Be the source of truth for customer and product data._ Traditionally, as companies scale their data warehouse becomes the source of truth and non-warehouse native tools (like product analytics) become less relevant as people lose trust in the data in them. However, by providing the data and data-intense tools in one place, we can enhance the power of our products (like product analytics), provide increased trust, _and_ enable companies to build on top of the warehouse itself as they see fit, all _without_ them having to setup a complex stack.
```

Reply:
one thing we need to clarify is exactly what comes in with the 80:20 rule for our focus stage customers - i think you're probably right so i'll go with your suggestion for now but we should be open minded I think?

Reply:
agreed, it'd be weird for our target customers to send employee data in as per your example